From a763610719e7d7f6cdc45569b6fbfdb91bb7c87b Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 27 Jul 2019 14:30:08 -0700
Subject: [PATCH] Do not include syscrtl.h with glibc

glibc 2.30 has deprecated it see [1]
Fixes
sys/sysctl.h:21:2: error: "The <sys/sysctl.h> header is deprecated and will be removed." [-Werror,-W#warnings]

[1] https://sourceware.org/git/?p=glibc.git;a=commit;h=744e829637162bb7d5029632aacf341c64b86990

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/helper/options.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/helper/options.c b/src/helper/options.c
index 12755e010..4007e37f6 100644
--- a/src/helper/options.c
+++ b/src/helper/options.c
@@ -34,7 +34,7 @@
 #if IS_DARWIN
 #include <libproc.h>
 #endif
-#ifdef HAVE_SYS_SYSCTL_H
+#if defined(HAVE_SYS_SYSCTL_H) && !defined(__GLIBC__)
 #include <sys/sysctl.h>
 #endif
 #if IS_WIN32 && !IS_CYGWIN
-- 
2.22.0

